docs/README.md

```diff
@@ -0,0 +1,69 @@
+# Kite for Visual Studio Code
+
+This extensions adds support for Kite in Visual Studio Code, including:
+
+- Completion Lists
+- Signature Help
+- Quick Info
+- Goto Definition 
+
+## How to use this extension
+
+This extension is automatically installed by Kite when you select Visual Studio Code in the list of plugins.
+
+### Startup
+
+When starting VSCode with Kite's plugin for the first time, the Kite's tour will be displayed in the active pane.
+
+![Kite Tour](./images/kite-tour.png)
+
+This tour will only be displayed once, if you want to see it again on next startup you can activate the `kite.showTourOnStartup` setting.
+
+### Status Bar
+
+The Kite icon in the status bar displays the state of Kite for the current file. By clicking on the icon you open the status panel with additional information.
```

Comment:
Change `By clicking on the icon you open the status panel with additional information.` → `Clicking on the icon will open the status panel with additional information.`